### DISCUSSION TITLE
Giving a user role Channel Access does not allow them to change entry status field

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#3878] [Feature Requests]
**Description of the problem**
User roles created that limit user access to some channels. Separately, the role also has Channel Manager access disabled.
These users cannot then attach (already defined) custom status values to entries.
This seems like an unintended constraint on the user.

**How To Reproduce**
As above.

**Error Messages**
None

**Environment Details:**
 - Version: 7.3.12
 - PHP Version 8.1
 - 
**Possible Solution**
If the user role is also given permission to access Channel Manager and then the option to create / edit / delete statuses, they can then also set custom status values on an entry.

**Additional context**

**Comment by intoeetive** [comment]
Statuses permission are a separate thing that can be edited by clicking pencil next to status name on Statuses tab in preferences for each channel (yes, I wish this would be more unified)

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/752126/25e2b1d4-39ee-409b-b343-d38a52090955)

Normally, when new Role is created it gets permissions to all statuses that exist. Same way, when new status is created, it get assigned to all Roles with CP access.

However in case of editing role that did not have CP access before I think you might end up with not all statuses being assigned / available. Which I think might be expected, it's just not intuitive on where to edit this

Classifying this as UX feature request
